src/js/events.js

```diff
@@ -21,7 +21,8 @@ function EventEmitter() {
   this._events = {};
 }
 
-module.exports.EventEmitter = EventEmitter;
+module.exports = EventEmitter;
```

Comment:
Based on the current NodeJs api, this is the preferred way to access the EventEmmiter. Thus we should also update the documentation. (And maybe the internal module also in a later PR)

Reply:
Good point, I forgot the documentation.